dags/atd_knack_markings_attachments.py

```diff
@@ -80,11 +83,11 @@
     t2 = DockerOperator(
         task_id="atd_knack_markings_attachments_to_agol",
         image=docker_image,
+        docker_conn_id="docker_default",
         auto_remove=True,
         command=f"./atd-knack-services/services/records_to_agol.py -a {app_name} -c {container} {date_filter_arg}",
         environment=env_vars,
         tty=True,
-        force_pull=True,
```

Comment:
i guess we don't need to force pull bc it's the same image that gets force pulled above 👍